pyenvbuilder/utils.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import subprocess
+from pathlib import Path
+
+
+logger = logging.getLogger(__name__)
+
+
+def validate_installed(args):
```

Comment:
What is args in this case?
Could we switch this to use something like: `shutil.which` instead of trying to run the command?
See: https://docs.python.org/3.3/library/shutil.html#shutil.which

Reply:
For the docstrings, we like Numpy-doc style and it would be good to add the parameters and returns such as:

```
Invokes the args and returns True if successful

Parameters
-----------
args : str
    Command to be tested

Returns
--------
tuple
     Boolean status and string status message

Reply:
I switched to using `shutil.which` - thank you for the recommendation. 
And fixed the docstrings to **Numpy-doc style**